frontend/pages/SoftwarePage/components/cards/SoftwareDetailsSummary/SoftwareDetailsSummary.tsx

```diff
@@ -122,17 +126,52 @@ const SoftwareDetailsSummary = ({
                 <TooltipTruncatedText value={displayName} />
               )}
             </h1>
-            {onClickEditIcon && (
-              <div className={`${baseClass}__edit-icon`}>
-                <Button
-                  onClick={onClickEditIcon}
-                  className={`${baseClass}__edit-icon-btn`}
-                  variant="icon"
-                >
-                  <Icon name="pencil" />
-                </Button>
-              </div>
-            )}
+            {onClickEditIcon &&
+              (() => {
+                const button = (
+                  <div className={`${baseClass}__edit-icon`}>
+                    <Button
+                      onClick={onClickEditIcon}
+                      className={`${baseClass}__edit-icon-btn`}
+                      variant="icon"
+                      disabled={gitOpsModeEnabled}
+                    >
+                      <Icon name="pencil" />
+                    </Button>
+                  </div>
+                );
+
+                if (gitOpsModeEnabled) {
+                  const tooltipContent = (
+                    <>
+                      {repoURL && (
+                        <>
+                          Manage in{" "}
+                          <CustomLink
+                            newTab
+                            text="YAML"
+                            variant="tooltip-link"
+                            url={repoURL}
+                          />
+                          <br />
+                        </>
+                      )}
+                      (GitOps mode enabled)
+                    </>
+                  );
+
+                  return (
+                    <TooltipWrapper
+                      tipContent={tooltipContent}
+                      underline={false}
+                    >
+                      {button}
+                    </TooltipWrapper>
+                  );
+                }
+
+                return button;
+              })()}
```

Comment:
We have the `GitOpsModeTooltipWrapper` component for this. Believe that's what you want here, and that'll allow you to not have to import the repo URL etc.